addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/channel-groups.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Channel Groups -->
+
+    <channel-group-type id="zone1Controls">
```

Comment:
Well, currently zone groups don't have any differences, but from spec point of view there are differences. Zone 1 (main) have most features, zone 2 doesn't have all zone 1 functions and zone 3 is most limited. So if/when zone specific functions will be added, zone specific groups is needed as well.